### DISCUSSION TITLE
How to construct a URL to firebase page given event_id?

### DISCUSSION THREAD

**Discussion by jhollida24** [firebase/firebase-ios-sdk#7888] [Q&A]
I know this is probably not the perfect venue for this, but it's not totally unrelated...

I'm using BigQuery to query large crashes that we have and do some more custom grouping. I would like to construct a URL to the Firebase web console page for crashes in these groups. The unique identifier for a crash instance seems to be event_id.

Is there a URL template I could plug event_id into? The URL for pages as you look through crash instances in an issue is something like:

https://console.firebase.google.com/u/0/project/project-name/crashlytics/app/ios:com.app.name/issues/{issue_id}?time=last-seven-days&sessionEventKey={mystery_value}_{event_id}

sessionEventKey appears to be a concatenation of some other value and event_id. What is this value? Is there a better way than this to get at a specific crash instance?

**Comment by morganchen12** [comment]
@samedson do you know if this is a stable identifier? My guess is the console team might decide to change it at any time and we don't want people relying on it being a consistent value.

**Reply by jhollida24** [reply]
I'm open to any other method of getting a console URL for the specific crash event. This was my very rough reverse engineering of the current URL scheme of that page.
